src/dowel/csv_output.py

```diff
@@ -42,38 +41,71 @@ def record(self, data, prefix=''):
                 self._writer.writeheader()
 
             if to_csv.keys() != self._fieldnames:
-                self._warn('Inconsistent TabularInput keys detected. '
```

Comment:
Since this was the last call to this method, is there any other code you can delete now?

Reply:
I don't think so. I removed everything related to CsvOutputWarning. The warnings for other classes should remain